client/components/community-translator/translatable-field.jsx

```diff
@@ -0,0 +1,22 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const TranslatableField = ( { originalString, title, fieldName, onChange, value, disabled } ) => (
```

Comment:
After speaking with @akirk, it's probably a good idea to rename this component to something more obvious. 
`<TranslationTextArea />` ? 
I'll leave it to your good judgement :)

Reply:
I think that's a good idea :)